### PR TITLE
Release Notes for  UgandaEMR 3.0.4

### PR DESCRIPTION
**Pull request by solemabrothers**

### CODE REVIEW

File: ugandaemr-304.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Over Due for third DNS PCR
+    * Updated Criteria for Due and Over Due for Second DNA PCR
+ 2. HIV MER Indicator Reports 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnantal Register
+
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers remove unnecessary whitespace

File: ugandaemr-304.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ 2. HIV MER Indicator Reports 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnantal Register
+    
+    
+  
+### Fixes Made
+
+       1. Disabled the creation of Multiple encounters on the same visit date
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers This indentation is not necessary

File: ugandaemr-304.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+       2. Reports Module does not start even after starting the module
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers provide root cause and how it was resolved

File: ugandaemr-304.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. System does not support to measure the number of patients seen by a perticular provider
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers Rewrite to Resolved issue with Data Entry Statistics that was ...

File: ugandaemr-304.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+       5.Clinical Assessment Forms are innaccessible from the HTML form entry section
+       6. Duplicate Clients in queue
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers what does this mean how were the duplicate clients being added

File: ugandaemr-304.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+* 32-bit installer -https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr3-0-4-installer-32bit.exe/download)
+* 64- bit installer -https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr3-0-4-installer-64bit.exe/download)
+
+#### WAR File
+
+[https://sourceforge.net/projects/ugandaemr/files/3.0.4/openmrs.war/download](https://sourceforge.net/projects/ugandaemr/files/3.0.4/openmrs.war/download)
+
+#### Upgrade Installer
+
+**For existing versions of UgandaEMR 2.0.\* to 3.0.4**
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers what is the \* for?

File: ugandaemr-304.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+* 64-bit upgrade files- https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr_upgrade_from_2.0.x_to_3.0.4_64bit.exe/download
+
+**For existing versions of UgandaEMR 1.x  to  3.0.4**
+
+* 32-bit upgrade files - https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr_upgrade_from_1.x_to_3.0.4_32bit.exe/download
+* 64-bit upgrade files - https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr_upgrade_from_1.x_to_3.0.4_64bit.exe/download
+
+### Fixing Failed Installations and Upgrades
+
+TBD
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers replace with none

**Comment by ssmusoke** [comment]
@solemabrothers Add a link to the SUMMARY.md page so that it appears in the left hand list

File: ugandaemr-304.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
+## UgandaEMR 3.0.4 Release Notes
+
+Currently, work in progress
```
Comment (ebichete) [inline ugandaemr-304.md]: Remove this "WIP" line

File: ugandaemr-304.md (hunk 6 above)
Comment (solemabrothers) [reply ugandaemr-304.md]: @ssmusoke we have two different versions i.e, 2.0.0 and 2.0.1, the * is to represent any of the versions

File: ugandaemr-304.md (hunk 7 above)
Comment (solemabrothers) [reply ugandaemr-304.md]: Done

File: ugandaemr-304.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Over Due for third DNS PCR
```
Comment (ebichete) [inline ugandaemr-304.md]: "Overdue" is one word.

File: ugandaemr-304.md (hunk 3 above)
Comment (ebichete) [reply ugandaemr-304.md]: "after starting" => "after manually starting"

**Comment by solemabrothers** [comment]
> 
> 
> @solemabrothers Add a link to the SUMMARY.md page so that it appears in the left hand list

Done

File: ugandaemr-304.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+    
+    
+  
+### Fixes Made
+
+       1. Disabled the creation of Multiple encounters on the same visit date
+       2. Reports Module does not start even after starting the module
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. System does not support to measure the number of patients seen by a perticular provider
+       5.Clinical Assessment Forms are innaccessible from the HTML form entry section
```
Comment (ebichete) [inline ugandaemr-304.md]: "innaccessible" => "inaccessible"

File: ugandaemr-304.md (hunk 2 above)
Comment (solemabrothers) [reply ugandaemr-304.md]: Done

File: ugandaemr-304.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+       6. Duplicate Clients in queue
+
+### Links to download files
+
+#### New Installation
+
+For new machines with no UgandaEMR installed
+
+* 32-bit installer -https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr3-0-4-installer-32bit.exe/download)
+* 64- bit installer -https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr3-0-4-installer-64bit.exe/download)
```
Comment (ebichete) [inline ugandaemr-304.md]: "64- bit" => "64-bit"

File: ugandaemr-304.md (hunk 9 above)
Comment (ebichete) [reply ugandaemr-304.md]: "DNS PCR" => "DNA PCR"

File: ugandaemr-304.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Over Due for third DNS PCR
+    * Updated Criteria for Due and Over Due for Second DNA PCR
+ 2. HIV MER Indicator Reports 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnantal Register
```
Comment (ebichete) [inline ugandaemr-304.md]: "Postnantal" => "Postnatal"

File: ugandaemr-304.md (hunk 2 above)
Comment (ebichete) [inline ugandaemr-304.md]: "Disabled" => "Prevent"

**Review by ebichete — CHANGES_REQUESTED** [review]

File: ugandaemr-304.md (hunk 6 above)
Comment (ebichete) [reply ugandaemr-304.md]: Use 2.0.x, similar to what we use below

File: ugandaemr-304.md (hunk 9 above)
Comment (solemabrothers) [reply ugandaemr-304.md]: Done

File: ugandaemr-304.md (hunk 12 above)
Comment (solemabrothers) [reply ugandaemr-304.md]: Done

File: ugandaemr-304.md (hunk 1 above)
Comment (solemabrothers) [reply ugandaemr-304.md]: Done

Comment (ssmusoke) [reply ugandaemr-304.md]: @solemabrothers Not done there are 3 empty lines here

File: ugandaemr-304.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
+## UgandaEMR 3.0.4 Release Notes
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Overdue for third DNA PCR
+    * Updated Criteria for Due and Overdue for Second DNA PCR
+ 2. HIV MER Indicator Reports 2.4
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers remove extra blank spaces at the front of the numbers they have an effect in markdown

File: ugandaemr-304.md [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+    * Overdue for third DNA PCR
+    * Updated Criteria for Due and Overdue for Second DNA PCR
+ 2. HIV MER Indicator Reports 2.4 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnatal Register
+    
+    
+  
+### Fixes Made
```
Comment (ssmusoke) [inline ugandaemr-304.md:12]: @solemabrothers Remove all the spaces before the numbers in this section, all should begin not indented

File: ugandaemr-304.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+       1. Prevent the creation of Multiple encounters on the same visit date
+       2. Reports Module does not start even after manually starting the module due to a duplicate uuid in the ugandaemrsync module
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. Resolved issue with Data Entry Statistics that was preventing the measure of patients seen by a particular provider
+       5. Clinical Assessment Forms are inaccessible from the HTML form entry section
+       6. Clients' names appearing more than once in the clinician queue
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers #6 should read like - Checking a patient into a location multiple times does not add the patient more than once in the queue

File: ugandaemr-304.md [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+    * Postnatal Register
+    
+    
+  
+### Fixes Made
+       1. Prevent the creation of Multiple encounters on the same visit date
+       2. Reports Module does not start even after manually starting the module due to a duplicate uuid in the ugandaemrsync module
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. Resolved issue with Data Entry Statistics that was preventing the measure of patients seen by a particular provider
+       5. Clinical Assessment Forms are inaccessible from the HTML form entry section
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers There is only one Clinical Assessment form

File: ugandaemr-304.md [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+ 3. Data Extraction Reports
+    * Postnatal Register
+    
+    
+  
+### Fixes Made
+       1. Prevent the creation of Multiple encounters on the same visit date
+       2. Reports Module does not start even after manually starting the module due to a duplicate uuid in the ugandaemrsync module
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. Resolved issue with Data Entry Statistics that was preventing the measure of patients seen by a particular provider
```
Comment (ssmusoke) [inline ugandaemr-304.md]: @solemabrothers Instead of "measure of patients" use "count of patients"
